### ISSUE TITLE
Error: No tag TODAY

### ISSUE THREAD

**Issue by wendyong92** [johannesjo/super-productivity#701]
### Steps to Reproduce

1. Shutdown laptop
2. Loads the app
3. Error occurs while loading
4. The app loads at the background while the error message is showing up as a persistent pop up box

P.S. I have no experience with github or json. Will greatly appreciate if you can guide me step by step on how to resolve this. Thanks!

### Stacktrace
```
webpack:///src/app/features/tag/store/tag.reducer.ts:46:12
function (webpack:///node_modules/@ngrx/store/fesm2015/ngrx-store.js:1096:29)
memoized (webpack:///node_modules/@ngrx/store/fesm2015/ngrx-store.js:1011:38)
args (webpack:///node_modules/@ngrx/store/fesm2015/ngrx-store.js:1059:38)
function (webpack:///node_modules/@ngrx/store/fesm2015/ngrx-store.js:1105:35)
memoized (webpack:///node_modules/@ngrx/store/fesm2015/ngrx-store.js:1011:38)
source$.pipe(pluck (webpack:///node_modules/@ngrx/store/fesm2015/ngrx-store.js:868:24)
_next (webpack:///node_modules/rxjs/_esm2015/internal/operators/map.js:29:34)
next (webpack:///node_modules/rxjs/_esm2015/internal/Subscriber.js:49:17)
super._subscribe (webpack:///node_modules/rxjs/_esm2015/internal/BehaviorSubject.js:14:23)
```

### Meta Info
META: SP6.1.0 Electron – en-US – Win32 – Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) superProductivity/6.1.0 Chrome/85.0.4183.121 Electron/10.1.5 Safari/537.36

### Actions Before Error
```
1606706123569: [Persistence] Save to DB 
1606706124501: [Task] Add time spent 
1606706124505: [Project] Update Work End 
1606706124522: [Tag] Update Work End for Tag 
1606706124526: [Persistence] Save to DB 
1606706124528: [Persistence] Save to DB 
1606706124530: [Persistence] Save to DB 
1606706125497: [Task] Add time spent 
1606706125503: [Project] Update Work End 
1606706125522: [Tag] Update Work End for Tag 
1606706125525: [Persistence] Save to DB 
1606706125527: [Persistence] Save to DB 
1606706125529: [Persistence] Save to DB 
1606706126509: [Task] Add time spent 
1606706126516: [Project] Update Work End 
1606706126540: [Tag] Update Work End for Tag 
1606706126544: [Persistence] Save to DB 
1606706126548: [Persistence] Save to DB 
1606706126551: [Persistence] Save to DB 
1606706127498: [Task] Add time spent 
1606706127504: [Project] Update Work End 
1606706127519: [Tag] Update Work End for Tag 
1606706127521: [Persistence] Save to DB 
1606706127524: [Persistence] Save to DB 
1606706127527: [Persistence] Save to DB 
1606706127824: [Task] Toggle start 
1606706127828: [Persistence] Save to DB 
1606706127829: [Task] UnsetCurrentTask 
1606706172167: [SP_ALL] Load(import) all data 
1606706172217: [SP_ALL] All Data was loaded
```

**Comment by johannesjo** [comment]
Thanks for reporting! This should already be fixed (#615) and it's basically a rare race condition causing this. The fix is only available on the edge channel currently. I'll try to provide a new release later this week.

**Comment by wendyong92** [comment]
Thanks!
